trino-aws-proxy-spi/src/main/java/io/trino/aws/proxy/spi/credentials/StandardIdentity.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.aws.proxy.spi.credentials;
+
+import static java.util.Objects.requireNonNull;
+
+public record StandardIdentity(String user)
```

Comment:
Should we add `List<T> groups` as well ? Its very probable that most use groups

Reply:
Having a type parameter is a problem. If `List<String>` works we can add that.

Reply:
+1 for `List<String>` groups